route-rs-runtime/src/processor/file_log.rs

```diff
@@ -0,0 +1,78 @@
+use crate::processor::{Log, Processor};
+use std::fmt::Debug;
+use std::fs::File;
+use std::marker::PhantomData;
+
+/// Wraps Log processor with a simpler File-specific interface
```

Comment:
Nit: Add more documentation here. Maybe something about how the file will be created, not appended to, can't exist already, etc.  Also talk about how we use buffered writes to keep it fast? Just some suggestions.

Reply:
I'll make a short follow up change to add some better docs. It'd be nice to explain why we're using `BufWriter` since it seems like a great abstraction to use here.